### PR TITLE
[feature] Fix trainable attribute callbacks

### PR DESCRIPTION
**Pull request by JPXKQX**
Fix node & edge trainable attribute callbacks. Callbacks are split into 2 classes:
- GraphNodeTrainableFeaturesPlot
- GraphEdgeTrainableFeaturesPlot

### CODE REVIEW

File: src/anemoi/training/diagnostics/callbacks/plotting.py
Comment (HCookie) [inline src/anemoi/training/diagnostics/callbacks/plotting.py:1]: This file will need to be renamed to `plot.py` in accordance with the latest changes in the base branch

File: src/anemoi/training/diagnostics/callbacks/__init__.py [hunk 1]
```diff
@@ -37,7 +37,13 @@
             plotting.PlotSample,
         ],
     ),
-    ("diagnostics.plot.learned_features", plotting.GraphTrainableFeaturesPlot),
+    (
+        "diagnostics.plot.learned_features",
```
Comment (HCookie) [inline src/anemoi/training/diagnostics/callbacks/__init__.py:41]: The key of `diagnostics.plot.learned_features` has been put in a deprecated dict in the latest changes to the base branch. This change can be incorporated into that. The recommended way to add these plots is through the `callbacks` list in the config

File: src/anemoi/training/config/diagnostics/eval_rollout.yaml
Comment (HCookie) [inline src/anemoi/training/config/diagnostics/eval_rollout.yaml:1]: This file will need to be rebased against the base branch

**Review by HCookie — CHANGES_REQUESTED** [review]
A couple changes needed to align with recent updates, then should be good to merge

File: src/anemoi/training/diagnostics/callbacks/__init__.py (hunk 1 above)
Comment (JPXKQX) [reply src/anemoi/training/diagnostics/callbacks/__init__.py:78]: Added!

**Comment by JPXKQX** [comment]
The latest changes to fxi/refactor_callbacks have been rebased now.

**Review by HCookie — APPROVED** [review]

**Pull request merged**